changehc/delphi_changehc/run.py

```diff
@@ -180,12 +181,21 @@ def run_module(params: Dict[str, Dict[str, Any]]):
                 elif numtype == "cli":
                     data = load_cli_data(file_dict["denom"],file_dict["flu"],file_dict["mixed"],
                              file_dict["flu_like"],file_dict["covid_like"],dropdate_dt,"fips")
-                su_inst.update_sensor(
+                more_stats = su_inst.update_sensor(
                     data,
                     params["common"]["export_dir"]
                 )
+                stats.extend(more_stats)
+
             logger.info("finished processing", geo = geo)
 
     elapsed_time_in_seconds = round(time.time() - start_time, 2)
+    min_max_date = stats and max(s[0] for s in stats)
```

Comment:
Ignoring the above comment for a moment, is min_max_date supposed to be the smallest max date? e.g. `min(max(s) for s in states)`

EDIT: nevermind I see it now.

Reply:
🤦‍♀️